### DISCUSSION TITLE
has anyone try to use h2 intead of derby

### DISCUSSION THREAD

**Discussion by ourlang** [nextgenhealthcare/connect#6229] [General]
Is there an example of which package configuration files need to be replaced

**Comment by jonbartels** [comment]
https://docs.nextgen.com/bundle/Mirth_User_Guide_40/page/connect/connect/topics/t_editing_the_properties_file_directly_windows_linux_mac_connect_ug.html 

A good search to run would be "postgres" or "mysql". Most people switch from Derby to PG, MS SQL, or MySQL. The concepts to change from one DB to another are always the same.

**Comment by ourlang** [comment]
Can't download this file? Is there a backup website available
Because I need H2 DATABASE

I am aware of the modification configuration of SQL Server and MySQL

https://forums.mirthproject.io/forum/mirth-connect/development/1956-has-anyone-try-to-use-h2-intead-of-derby

**Comment by ourlang** [comment]
http://www.mirthproject.org/community/forums/fb_attachments/Mirth-e01c69464d1abc345d723e6555e68a15.zip

**Comment by ourlang** [comment]
![image](https://github.com/nextgenhealthcare/connect/assets/32355185/f8c0d4d0-279a-4393-8dfd-d59f5296f588)

**Comment by ourlang** [comment]
I replaced some of them, but there were some errors

**Comment by ourlang** [comment]
h2 resolved

**Reply by pacmano1** [reply]
Please share your solution.

**Comment by ourlang** [comment]
[h2 intead of derby2024.zip](https://github.com/user-attachments/files/15873226/h2.intead.of.derby2024.zip)
